src/components/GoodItem/GoodItem.tsx

```diff
@@ -0,0 +1,14 @@
+type Props = {
+  good: string,
+};
+
+export const GoodsItem: React.FC<Props> = ({ good }) => {
+  return (
+    <li
+      key={good}
+      data-cy="Good"
+    >
+      {good}
+    </li>
+  );
```

Comment:
Better to avoid separating `ul`s and `li`s, also `key` attribute should be specified in place in code where array is mapping